web/src/app/about/AboutContainer.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import AboutComponent from './AboutComponent';
+
+class AboutContainer extends Component {
+
```

Comment:
be consistent with spacing. AboutContainer has a blank line between 
Component {...

render(..

but AboutComponent does not.